Src/FluentAssertions/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -40,6 +39,17 @@ public EquivalencyAssertionOptions<TExpectation> Excluding(Expression<Func<TExpe
             return this;
         }
 
+        /// <summary>
+        /// Excludes the specified (nested) member from the structural equality check.
```

Comment:
🤔 I don't think this covers what `For` does, right? It's not that it excludes the specified property, but just identifies the collection that you want to exclude a property from, isn't it?

Reply:
Updated the description 👍